base_layer/transactions/src/proto/types_impls.rs

```diff
@@ -79,3 +79,19 @@ impl From<HashOutput> for proto::HashOutput {
         Self { data: output }
     }
 }
+
+//--------------------------------- BlindingFactor -----------------------------------------//
+
+impl TryFrom<proto::BlindingFactor> for BlindingFactor {
+    type Error = ByteArrayError;
+
+    fn try_from(offset: proto::BlindingFactor) -> Result<Self, Self::Error> {
+        Ok(RistrettoSecretKey::from_bytes(&offset.data)?)
```

Comment:
The idea of using `types` is to limit an import dependency on RistrettoSecretKey to exactly one place. `BlindingFactor::from_bytes` should work here. And then you can drop the `use RistrettoSecretKey`. 

Ditto for RistrettoPublicKey

Reply:
Made use of `BlindingFactor` instead and removed the use of `RistrettoSecretKey` and `RistrettoPublicKey`. Also corrected it for the`Signature` proto conversions.

Reply:
👍